webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-pre/hook-template

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Use 'HOOK_WARN_TIMEOUT' or 'HOOK_KILL_TIMEOUT' *ABOVE THAT COMMENT* to specify a time after which a warning is emitted
+# or the hook is interrupted.
+# For example, 'HOOK_WARN_TIMEOUT' = 1 second (without quote, without text on the line after duration) will emit a warning if
+# the hook takes more than one second to run.
+
+# Parameters are passed by environment variable:
+#
+# - RUDDER_GENERATION_DATETIME: generation datetime: ISO-8601 YYYY-MM-ddTHH:mm:ss.sssZ date/time that identify that policy generation.
+
+# Errors code on hooks are interpreted as follow:
+# - 0     : success, no log (apart if debug one)          , continue to next hook
+# - 1-31  : error  , error   log in /var/log/rudder/webapp/, stop processing
```

Comment:
this need to be adapted. 

I think "stop processing" should be updated in all hook template to be more precise on what is stop (ie just processing of other hooks) ; and on that one, very loudly say that it will prevent policy generation (and so possibly block all policy generation in case of error in the script)